crypto/evp/e_aes.c

```diff
@@ -2931,8 +2937,10 @@ static int aes_gcm_ctrl(EVP_CIPHER_CTX *c, int type, int arg, void *ptr)
                 gctx_out->iv = EVP_CIPHER_CTX_iv_noconst(out);
             else {
                 gctx_out->iv = OPENSSL_malloc(gctx->ivlen);
-                if (gctx_out->iv == NULL)
+                if (gctx_out->iv == NULL) {
+                    EVPerr(EVP_F_AES_GCM_CTRL, ERR_R_MALLOC_FAILURE);
```

Comment:
Hmm.. in this function there are also a lot more not obvious error conditions without EVPerr.

Reply:
Once started it‘s hard to stop again... ;-)